scripts/polkadot-launch/README.md

```diff
@@ -79,6 +79,12 @@ Need to do to run 4 relay chain nodes, 2 Composable's collators and 2 Basilisk's
 	yarn composable_and_basilisk
 	```
 
+or you can use Docker image
+
+	```bash
+	docker run --rm -ti -u$(id -u):$(id -g) -p9944:9944 -p9988:9988 -p9998:9998 composablefi/composable-sandbox:latest
```

Comment:
may be remove copy paste from here and force people to use make (avoid copy paste)

Reply:
Yes, but it is required to install cargo-make, Rust and etc.

Reply:
but make requires install make and docker docker. do you promise to support 2 versions of run command up to date i 2 places?